workspaces/ui/src/contexts/CaptureContext.js

```diff
@@ -75,7 +75,7 @@ class _CaptureContextStore extends React.Component {
           if (this.state.notificationChannel) {
             this.state.notificationChannel.close();
           }
-          // track('Completed Diff', {
+          // track('Completed IAsyncTask', {
```

Comment:
Not sure if you left this intentionally, but might be a bit of revert back from `AsyncTask` -> `Diff` missed.